app/controllers/signup.js

```diff
@@ -189,10 +188,15 @@ export default class SignupController extends Controller {
     // submit
     // https://github.com/Real-Dev-Squad/website-api-contracts/tree/main/users#patch-usersself
     e.preventDefault();
+    let reqBody = this.formData;
+    if (!this.formData.website) {
```

Comment:
Is website the _only_ optional field? Can we potentially have other fields that we don't need to be sending to our backend? e.g Instagram?

Will it make sense to create a function `deleteEmptyOptionalFields` that ensures that ?